modules/rosa-create-objects.adoc

```diff
@@ -204,6 +204,15 @@ a|--cluster-name <cluster_name>
 |--controlplane-iam-role <arn>
 |The ARN of the IAM role to attach to control plane instances.
 
+|--create-cluster-admin
+|Optional. As part of cluster creation, create a local administrator user (`cluster-admin`) for your cluster. This automatically configures an htpasswd identity provider for the `cluster-admin` user. Optionally, use the `--cluster-admin-user` and `--cluster-admin-password` options to specify the username and password for the administrator user. Omitting these options automatically generates the credentials and displays their values as terminal output.
```

Comment:
```suggestion
|Optional parameter. As part of cluster creation, create a local administrator user (`cluster-admin`) for your cluster. This automatically configures an `htpasswd` identity provider for the `cluster-admin` user. Optionally, use the `--cluster-admin-user` and `--cluster-admin-password` options to specify the username and password for the administrator user. Omitting these options automatically generates the credentials and displays their values as terminal output.
```

Reply:
This doesn't match the rest of the documentation; if we need to make this change, I'd like to do it globally.